theme/templates/accounts/profile.html

```diff
@@ -4,78 +4,521 @@
-<div class="col-md-4">
-<h2>Resources</h2>
-    <ul class="nav nav-stacked">
-    {% for res in resources %}
-        <li><a href="{{ res.get_absolute_url }}">{{ res.metadata.title }}</a></li>
-    {% empty %}
-        <li>This user has no Resources, yet.</li>
-    {% endfor %}
-    </ul>
-</div>
-</div>
-
-
-{% endwith %}
+                                            <span class="text-muted"> shared </span>
+                                            <span><a href="{{ res.get_absolute_url }}">{{ res.metadata.title }}</a></span>
+                                            <h5 style="margin:0 50px;" class="text-muted">Resource type: <strong>{{ res|resource_type }}</strong></h5>
+                                            <h5 style="margin:0 50px;margin-bottom:15px;" class="text-muted">Date: <strong>{{ res.created }}</strong></h5>
+
+                                            {% if res.metadata.description %}
+                                                <div class="activity-description col-sm-12"
+                                                        {% if profile.picture and profile.picture.url %}style="margin-left:15px;" {% endif %}>
+                                                    <p class="description">
+                                                        <strong>ABSTRACT: </strong>{{ res.metadata.description|linebreaks }}
+                                                    </p>
+                                                </div>
+                                                <div class="show-more-btn btn-default" title="Show abstract">···</div>
+                                            {% endif %}
+                                        </div>
+                                        {% endif %}
+                                    {% endfor %}
+
+                                    {% else %}
+                                        <span class="text-muted">This user has no recent activity.</span>
+                                    {% endif %}
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="col-md-5">
+                            {#        DESCRIPTION       #}
+
+                            {% if profile.details and user.pk %}
+                            <div class="panel panel-default">
+                                <div class="panel-heading">
+                                    <h3 class="panel-title">About {{ u.first_name }}</h3>
+                                </div>
+                                <div class="panel-body" style="color:#757575;">
+                                    {{ profile.details }}
+                                </div>
+                            </div>
+                            {% endif %}
+
+                            {#  CONTACT #}
+                            {% if profile.public or u.id == user.id and user.pk %} {# Show only if profile is public and user is logged in #}
```

Comment:
Is `u` the current profile user or is `user`? Is one the profile and one the logged in user? If so that's terribly confusing and needs corrected or will lead to bad mixups in the future.